game/scripts/npc/abilities/bristleback_quill_spray.txt

```diff
@@ -42,7 +42,7 @@
       "02"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "quill_base_damage"                               "20.0 40.0 60.0 80.0 100.0 120.0"
+        "quill_base_damage"                               "20.0 40.0 60.0 80.0 120.0 160.0"
```

Comment:
If I wanted to buff Quill Spray, I would make `max_damage` scale better too. Because 550 is meh when you reach lvl 25 and everyone has armor and ton of hp regen.

Reply:
You are also 3 levels off 28 where you can level up Quill Spray. Problem with buffing the Quill Spray damage is if I buff it at Dota levels it makes level 7 BB better. Early bristle back is good.

Reply:
Early Bristleback is good only if you are not behind and if your team is winning.

Reply:
Max damage is 1100 2200 at OAA levels

Reply:
So you think `550 660 770 880 1100 2200` is too much?

Reply:
I did "550 600 650 700" For Now. Tbh You will have a hard time reaching that stack damage so I might look at buffing the level 20 talent if BB still sucks.